includes/polling/os/moxa-etherdevice.inc.php

```diff
@@ -10,12 +10,18 @@
  * the source code distribution for details.
  */
 
+// hardware => switchModel.0
+// version => firmwareVersion.0
+// serial => serialNumber.0 (not supported on all models)
 
-// Moxa people enjoy creating MIBs for each model!
-if ($device['sysDescr'] == 'IKS-6726A-2GTXSFP-T') {
-    $mibmod = 'MOXA-IKS6726A-MIB';
-} elseif ($device['sysDescr'] == 'EDS-G508E-T') {
-    $mibmod = 'MOXA-EDSG508E-MIB';
+$oids = array(
+    'hardware' => $device['sysObjectID'].'.1.2.0',
+    'version' => $device['sysObjectID'].'.1.4.0', 
+    'serial' => $device['sysObjectID'].'.1.78.0', 
+);
+  
+$os_data = snmp_get_multi_oid($device, $oids);
+
+foreach ($oids as $var => $oid) {
+    $$var = trim($os_data[$oid], '"');
 }
-$version = snmp_get($device, "firmwareVersion.0", "-OQvs", $mibmod);
```

Comment:
Can you change $oids to something else, it's quite a generic variable and could be defined elsewhere.

Also, unset() whichever name you use + $os_data

Reply:
Was trying to follow the style in other polling modules with the use of `$oids`. Commited `unset()` at the end.